src/main/resources/db/changelog/changes/create-users-roles-table.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-roles-table
+      author: Denys Diuimov
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    foreignKeyName: fk_users_roles_user
+                    references: users(id)
+              - column:
+                  name: role_id
+                  type: bigint
+                  constraints:
+                    foreignKeyName: fk_users_roles_role
```

Comment:
I also added constraint `primaryKey: true` for both, as these pairs shouldn't duplicate

Reply:
done